streams/src/main/java/org/apache/kafka/streams/processor/TimestampExtractor.java

```diff
@@ -20,7 +20,8 @@
 import org.apache.kafka.clients.consumer.ConsumerRecord;
 
 /**
- * An interface that allows the Kafka Streams framework to extract a timestamp from an instance of {@link ConsumerRecord}
+ * An interface that allows the Kafka Streams framework to extract a timestamp from an instance of {@link ConsumerRecord}.
+ * The extracted timestamp is defined as milliseconds.
```

Comment:
What we'd also need to specify is "milliseconds *since when*" -- I assume we want the semantics of `System.currentTimeMillis()` = Unix timestamp in milliseconds = milliseconds since the epoch?